backend/de.metas.adempiere.adempiere/base/src/main/java-legacy/org/compiere/model/MIssue.java

```diff
@@ -57,38 +51,9 @@ public MIssue(Properties ctx, ResultSet rs, String trxName)
 		super(ctx, rs, trxName);
 	}
 
-	private void init(final Properties ctx) throws Exception
+	private void init(final Properties ctx)
```

Comment:
consider deleting MIssue class at all... to avoid double work.

The only thing we need to take care in ADIssueFactory is truncating those long text columns...